cmd/juju/model/grantrevoke.go

```diff
@@ -418,7 +418,7 @@ func setUnsetUsers(c accountDetailsGetter, offerURLs []*crossmodel.OfferURL) err
 				return errors.Trace(err)
 			}
 		}
-		url.User = currentAccountDetails.User
+		url.ModelQualifier = currentAccountDetails.User
```

Comment:
Do we need to do any string manipulation here? A comment would be good.

Reply:
missed this one, fixed